src/main/java/nl/tudelft/context/controller/MenuController.java

```diff
@@ -35,79 +32,114 @@
      * Create a new menu.
      *
      * @param mainController The MainController of the application.
-     * @param menuBar The menubar this Menu should hook onto.
+     * @param menuBar        The menubar this Menu should hook onto.
      */
     public MenuController(final MainController mainController, final MenuBar menuBar) {
         this.mainController = mainController;
-        this.menuBar = menuBar;
         menuBar.setUseSystemMenuBar(true);
 
-        initFileMenu();
-        initNavigateMenu();
-        initHelpMenu();
+        menuBar.getMenus().add(createMenu("_File",
+                createWorkspaceLoader("Select Workspace Folder",
+                        new KeyCodeCombination(KeyCode.O, KeyCombination.CONTROL_DOWN)),
+                createMenuItem("Exit",
+                        event -> mainController.exitProgram())));
+        menuBar.getMenus().add(createMenu("_Navigate",
+                createMenuItem("Previous",
+                        new KeyCodeCombination(KeyCode.ESCAPE),
+                        event -> mainController.previousView()),
+                createMenuItem("Show Phylogenetic tree",
+                        new KeyCodeCombination(KeyCode.Q, KeyCombination.CONTROL_DOWN),
```

Comment:
I changed it to T + SUPER_DOWN (Windows: Ctrl + T / Mac: ⌘ + T). This, because Q + SUPER_DOWN (⌘ + Q) is a very common combination to shut down applications, somewhat similar to Alt + F4 on Windows. It does override the system's default binding, but this is bad practice.

It seems to me that the master hasn't been merged into this branch for a while. Could you do that first?